install-dev/upgrade/php/ps_1770_update_order_status_colors.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+use PrestaShop\PrestaShop\Core\Domain\Order\Status\OrderStatusColor;
+
+/**
+ * Updates order status colors according to new color schema
+ */
+function ps_1770_update_order_status_colors() {
+    $statusColorMap = [
+        OrderStatusColor::AWAITING_PAYMENT => Configuration::getMultiple([
```

Comment:
Why not using const defined in config.inc.php?

Reply:
Because of deprecation, see

https://github.com/PrestaShop/PrestaShop/blob/develop/config/config.inc.php#L242

Reply:
Oh, you're right, didn't saw it. Could we have class const at least? or is it too overkill?

Reply:
Hmmm, I guess it would make sense to have class constants for payment status config names, but it's beyond the scope of this PR imo.